Js/popup.js

```diff
@@ -0,0 +1,66 @@
+import popupdata from './data.js';
+
+const submit = document.querySelectorAll('.see-project');
+const popopcontainer = document.querySelector('.popup-container');
+const popupwindow = (data) => {
+  popopcontainer.innerHTML = ` <div class="container-popup">
```

Comment:
- [ ] Some content on your popup seems to be chopped off on both mobile and desktop views. I think you can fix this by removing the margin-bottom of the container ad setting the popup min-height to 100%

![image](https://github.com/alizaamir123/mobile-version/assets/63473584/37f7389b-5289-45c3-9c95-a11fdb9c340d)